app/import.go

```diff
@@ -45,6 +45,9 @@ func (a *App) BulkImport(fileReader io.Reader, dryRun bool, workers int) (*model
 	var linesChan chan LineImportWorkerData
 	lastLineType := ""
 
+	buf := make([]byte, 0, 64*1024)
+	scanner.Buffer(buf, 4096*4096)
```

Comment:
Let's rename this to a constant - `const maxScanTokenSize = 16*1024*1024 // Need to set a higher limit than default because some customers cross the limit. See MM-22314`.

Also, do this immediately after initializing the scanner. Because - `Buffer panics if it is called after scanning has started.` (https://golang.org/pkg/bufio/#Scanner.Buffer)

Reply:
Agree on the first comment but initializing `NewScanner` is simply creates a struct. The `Scan()` method starts the scanning therefore there were no risk :) Though, it would be more idiomatic to configure where it is created 👍